Makefile.am

```diff
@@ -5,7 +5,6 @@ all:
 	$(MAKE) -C cppForSwig
 
 clean-local:
-	$(MAKE) -C cppForSwig clean
```

Comment:
Any reason why this was removed? Is it just something you wanted to sneak in? Just trying to understand it.